tests/test_process_elife_zip_dag.py

```diff
@@ -91,20 +91,24 @@ def test_extract_archived_files_to_bucket_raises_exception_when_article_not_in_z
     assert str(error.value) == 'elife-00666.xml not in elife-00666-vor-r1.zip: []'
 
 
-def test_convert_tiff_images_in_expanded_bucket_to_jpeg_images_using_article_with_tiff_images(context, s3_client, mocker):
+@pytest.mark.parametrize('zip_file, uploaded_file, num_of_tiffs', [
+    ('elife-36842-vor-r3.zip', 'elife-36842-vor-r3/elife-36842-fig1.jpg', 25),
+    ('elife-40092-vor-r2.zip', 'elife-40092-vor-r2/elife-40092-fig1.jpg', 11)
+])
+def test_convert_tiff_images_in_expanded_bucket_to_jpeg_images_using_article_with_tiff_images(
+        zip_file, uploaded_file, num_of_tiffs, context, s3_client, mocker):
```

Comment:
`sample_uploaded_file` since its just one of 25?

Reply:
updated.